### DISCUSSION TITLE
请问如何生成 Dynamic 参数的Action?

### DISCUSSION THREAD

**Discussion by AshWang-whm** [dotnetcore/Natasha#120] [Q&A]
![image](https://user-images.githubusercontent.com/31700716/153321520-1ba3c5a4-1afd-45a3-abc2-3baca05758fb.png)
![image](https://user-images.githubusercontent.com/31700716/153321539-07af417d-3ca1-4cc2-93ac-15637386fc36.png)
请问如何生成 Dynamic 参数的Action? 似乎Dynamic 类型会被转成 Object 类型.

最终想要实现这种效果
![image](https://user-images.githubusercontent.com/31700716/153321895-873b8d6f-ade6-495e-b667-893e3b58d6b5.png)

**Comment by NMSAzulX** [comment]
已经用 Natasha 了 没必要用  dynamic 了. 另外 Dynamic 无法反射出来.  可以使用 FastMethodOperator 去手动构建方法.

**Reply by AshWang-whm** [reply]
好的,感谢回复.
